README.md

```diff
@@ -83,6 +86,49 @@ Once you have these you can run `make test`, passing your Sandbox credentials as
 PLAID_CLIENT_ID=aabbcc PLAID_PUBLIC_KEY=ddeeff PLAID_SECRET=ffeedd make test
 ```
 
+## Examples
+
+### Payment Initiation
+For more information about this product, head to the [Payment Initiation docs](https://plaid.com/docs/payment-initiation/).
+
+#### Create payment recipient using BACS with no IBAN or address
```

Comment:
should this content just be in https://plaid.com/docs/payment-initiation/add-to-app/#create-a-recipient ? We dont generally add documentation to this readme file

Reply:
Yeah FWIW my preference would be for any example code to be added as tests, rather than to the README.  That way we keep the README more concise, we're more consistent across products, and we get the benefit of test coverage

Reply:
Thanks for the comments @skylarmb @phoenixy1 

My thinking was that we have examples in README files for other clibs and they are a bit easier to read than unit tests. But I understand that is important to be consistent across products, so I'm closing this PR now.